db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -101,32 +142,54 @@ def collect_dialog_totals(prov_option, options_hash)
 end
 
 def dialog_values(prov_option, options_hash, dialog_array)
+  args_hash = {:prov_option => prov_option,
+               :array       => dialog_array,
+               :cloud       => false}
+
   options_hash.each do |_sequence_id, options|
-    set_requested_value(prov_option, options[prov_option], options[:instance_type], dialog_array)
+    args_hash[:prov_value] = options[prov_option]
+    args_hash[:flavor] = flavor_obj(options[:instance_type])
+    request_hash_value(args_hash)
   end
 end
 
-def set_requested_value(prov_option, option_value, find_id, dialog_array)
-  $evm.log(:info, "set requested value: prov_option: #{prov_option} value: #{option_value}")
-  return if cloud_value(prov_option, option_value, find_id, dialog_array)
+def request_hash_value(args_hash)
+  return if cloud_value(args_hash)
+
+  default_option(args_hash[:prov_value], args_hash[:array])
+end
+
+def cloud_storage(flavor, dialog_array)
+  return false unless flavor
+  storage = flavor.root_disk_size.to_i + flavor.ephemeral_disk_size.to_i + flavor.swap_disk_size.to_i
+  default_option(storage, dialog_array)
+  true
+end
 
-  $evm.log(:info, "set requested value default_option: prov_option: #{prov_option} value: #{option_value}")
-  default_option(option_value, dialog_array)
+def cloud_number_of_cpus(flavor, dialog_array)
+  return false unless flavor
+  $evm.log(:info, "Retrieving cloud flavor #{flavor.name} cpus => #{flavor.cpus}")
+  default_option(flavor.cpus, dialog_array)
+  true
```

Comment:
The return is no longer being used so it can be changed to not return boolean values.